client/src/components/mapping/FhirMappingPanel/FhirResourceTree/nodeLabel.tsx

```diff
@@ -1,31 +1,30 @@
 import React from 'react';
-import { Menu, MenuItem, ContextMenu } from '@blueprintjs/core';
+import { Menu, MenuItem, ContextMenu, Tooltip } from '@blueprintjs/core';
+import { Attribute } from '@arkhn/fhir.ts';
 
 interface NodeLabelProps {
-  name: string;
-  type: any;
+  attribute: Attribute;
   addNodeCallback: any;
   deleteNodeCallback: any;
 }
 
 export const NodeLabel = ({
-  name,
-  type,
+  attribute: { types, definition, id, isArray, isItem, isSlice },
   addNodeCallback,
   deleteNodeCallback
 }: NodeLabelProps) => {
   const showContextMenu = async (e: React.MouseEvent<HTMLDivElement>) => {
     e.preventDefault();
 
-    const menu = addNodeCallback ? (
+    const menu = isArray ? (
       <Menu>
         <MenuItem
           icon={'add'}
           onClick={addNodeCallback}
           text={'Ajouter un item'}
         />
       </Menu>
-    ) : deleteNodeCallback ? (
+    ) : isItem && !isSlice ? (
```

Comment:
sure. By definition, an array can have 1..N _items_ (not _children_). If the array have slices, slices are "propagated" (copied) to each children so that each item of the array can "implement" a slice.